.gen/nfpm.yml.tpl

```diff
@@ -11,3 +11,7 @@ contents: # contents to add to the package
 
   - src: "./{{ getenv "APPNAME" }}.yml" # local path of agent yml
     dst: "/etc/opt/srlinux/appmgr/{{ getenv "APPNAME" }}.yml" # destination path of agent yml
+
+  - src: "/opt/{{ getenv "APPNAME" }}/run.sh"
+    dst: "/usr/local/bin/{{ getenv "APPNAME" }}"
+    type: "symlink"
```

Comment:
Hi @sebastien-guay 
can you check an agent installed via rpm works? I tried it today on a hardware box and the app errored and the box went nuts =)

just build the rpm with `make rpm`, move it to the srlinux, do `rpm -U <rpmfile>` and then `/tools system application-management application appmgr reload` to add the app

Reply:
> Hi @sebastien-guay can you check an agent installed via rpm works? I tried it today on a hardware box and the app errored and the box went nuts =)
> 
> just build the rpm with `make rpm`, move it to the srlinux, do `rpm -U <rpmfile>` and then `/tools system application-management application appmgr reload` to add the app

Yes I did the rpm install on a SRL clab node before pushing the changes :(. I will take a look.

Reply:
> Hi @sebastien-guay can you check an agent installed via rpm works? I tried it today on a hardware box and the app errored and the box went nuts =)
> 
> just build the rpm with `make rpm`, move it to the srlinux, do `rpm -U <rpmfile>` and then `/tools system application-management application appmgr reload` to add the app

@hellt Installing with the RPM is working like when I tested before. You must do a `make build-app` to have the python environment in the `greeter' folder.` The rpm does not have the `.venv` if you only do `make rpm`.

Reply:
maybe it makes sense to add `build-app` as a dependency for `rpm` target? so that we won't let users build an RPM that is not complete?

Reply:
`make rpm` does what it says, it builds the rpm. `make build-app` call 'build-venv' and then 'rpm'.

Reply:
> 

I can remove the `rpm` task and just keep `build-app`. What do you think @hellt ?

Reply:
Let's do the other way around. make rpm should produce a deployable rpm package. This means any prerequisite steps should be called by it. 
This means that it should require build-app as it's dependency

At the end we should have common make targets synced between go and py versions

Reply:
> Let's do the other way around. make rpm should produce a deployable rpm package. This means any prerequisite steps should be called by it. This means that it should require build-app as it's dependency
> 
> At the end we should have common make targets synced between go and py versions

Currently `build-app` make sense for go but not for python since nothing is really built so `rpm` should have only a dependency on `build-venv`. `build-app` would make sense if we create an executable.

Reply:
Yes, I would like rpm to require any target that is needed so that it doesn't produce faulty package

Reply:
> Yes, I would like rpm to require any target that is needed so that it doesn't produce faulty package

@hellt Ready for you to take another look.